lib/locales/it.ts

```diff
@@ -264,6 +264,7 @@ export const locale: Locale = {
     clement_i_pope: 'San Clemente I, papa e martire',
     columba_of_iona_abbot: 'San Colombano, abate',
     columba_of_iona_abbot_copatron_of_ireland: 'San Colombano, abate e patrono secondario dell’Irlanda',
+    columban_of_luxeuil_abbot: 'San Colombano, abate', // src: missal_it_2020_ed3
```

Comment:
Just a nitpick: we might change this, however, everywhere I use `mr_${lang}_${year}_${edition}` format.

```suggestion
    columban_of_luxeuil_abbot: 'San Colombano, abate', // src: mr_it_2020_ed3
```